db/migrate/20240109223006_create_posts.rb

```diff
@@ -0,0 +1,13 @@
+class CreatePosts < ActiveRecord::Migration[7.1]
+  def change
+    create_table :posts do |t|
+      t.integer :author_id
```

Comment:
Well done on adding the `belongs_to` association in the model! However, for full data integrity and to optimize database performance, it's essential to establish relationships at the database level. Even though you've created associations in the model layer, it's highly recommended to reflect these changes in the database using migration files. This aligns with good database design practices and ensures that **The database accurately represents the structure of your ER diagram**.  👍 
- [ ] I encourage you to follow the advice from the previous review and update the migration file accordingly for all three tables including `posts`, `comments`, and `likes`. This approach will synchronize the model layer with the database, providing a comprehensive solution. 🙏 
- [ ] Additionally as mentioned in the project requirement to use migration file. I will highly recommend you follow the project requirements and update your code. 🙏 
![image](https://github.com/francksefu/blog_web/assets/55840999/db88cd3d-9164-4423-9ebc-3c7da3fb3bf8)

*Note: Uu will work with the association in the model layer in the next milestone.*

Keep up the good work! If you have any questions or need further clarification, feel free to reach out. 🚀🛠️